.github/workflows/build.yml

```diff
@@ -0,0 +1,272 @@
+        run: mvn clean install && mvn package -f App
+
+      - name: Create Bundle using jpackage
+        id: outputfile
+        run: |
+          .github/scripts/build_app_mac.sh
+          echo ::set-output name=path::App/target/installer/ChatApp-${{ env.APP_VERSION }}.pkg
+        env:
+          GLUON_MACSIGN_PREFIX: ${{ secrets.GLUON_MACSIGN_PREFIX }}
+          GLUON_MACSIGN_USER: ${{ secrets.GLUON_MACSIGN_USER }}
+          PROJECT_VERSION: 1.1.0-SNAPSHOT
+          APP_VERSION: 1.0.0
+
+      - name: Upload installer
+        uses: actions/upload-artifact@v2
+        with:
+          name: InstallerPackageMac
+          path: ${{ steps.outputfile.outputs.path }}
+
+
+  macos-native:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Gluon's GraalVM
+        uses: gluonhq/setup-graalvm@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Make staging directory
+        run: mkdir staging
+
+      - name: Gluon License
+        uses: gluonhq/gluon-build-license@master
+        with:
+          gluon-license: ${{ secrets.GLUON_LICENSE }}
+          dir: App
+
+      - name: Gluon Build
+        run: mvn clean install && mvn -Pdesktop gluonfx:build -f App
+
+      - name: Copy native image to staging
+        run: |
+          cp -r App/target/gluonfx/x86_64-darwin/ChatApp* staging
+          cp -r App/target/gluonfx/x86_64-darwin/gvm/*.json staging
+          cp -r App/target/gluonfx/x86_64-darwin/gvm/log/* staging
+          
+      - name: Upload native image
+        uses: actions/upload-artifact@v2
+        with:
+          name: NativePackageMac
+          path: staging
+
+
+  windows-jpackage:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+
+      - name: Visual Studio shell
+        uses: egor-tensin/vs-shell@v1
+
+      - name: Make staging directory
+        run: mkdir staging
+
+      - name: Gluon License
+        uses: gluonhq/gluon-build-license@master
+        with:
+          gluon-license: ${{ secrets.GLUON_LICENSE }}
+          dir: App
+
+      - name: Setup java
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Build and package
+        run: mvn clean install && mvn package -f App
+
+      - name: Create Bundle using jpackage
+        id: outputfile
+        shell: cmd
+        run: |
+          call .github\scripts\build_app_windows.bat
+          echo ::set-output name=path::App\target\installer\ChatApp-${{ env.APP_VERSION }}.msi
+        env:
+          PROJECT_VERSION: 1.1.0-SNAPSHOT
+          APP_VERSION: 1.0.0
+
+      - name: Upload installer
+        uses: actions/upload-artifact@v2
+        with:
+          name: InstallerPackageWindows
+          path: ${{ steps.outputfile.outputs.path }}
+
+
+
+  windows-native:
```

Comment:
this will work once we can run native-image. But so far it is not doing anything. Can we disable the job itself?

Reply:
done